package.json

```diff
@@ -292,15 +287,15 @@
     "eslint-plugin-prettier": "^3.4.0",
     "eslint-plugin-react": "^7.24.0",
     "eslint-plugin-react-hooks": "^4.2.0",
-    "husky": "~4.3.8",
-    "lint-staged": "^10.5.2",
+    "husky": "^7.0.0",
```

Comment:
Wow, that's a huge leap.

Reply:
In less than 6 months, they broke the lib in several majors because they changed it a lot